rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -329,6 +337,40 @@ def _decode_repay(
 
         return return_event, repay_event
 
+    def _decode_mint_events(self, context: DecoderContext) -> None:
+        """Decode possible mint events in transactions. Those are result of collateral/debt APY"""
```

Comment:
Link an example transaction in the docstring. You probably have one in the tests but do one here.

Also what do you mean by:

> Those are result of collateral/debt APY

Reply:
it was the interest